invenio_search/walkers/elasticsearch.py

```diff
@@ -123,6 +120,11 @@ def _f(keyword):
     @visitor(DoubleQuotedValue)
     def visit(self, node):
         def _f(keyword):
+            if keyword == [u'exactauthor.raw']:
+                from inspire.modules.authors.utils import author_tokenize
+                name_variations = author_tokenize(node.value)
+                return {"bool": {"must": [{"terms": {"minimum_match": 1, "authors.name_variations": name_variations}}, {
+                    "term": {"_collections": "HEP"}}], "should": [{"match": {"authors.full_name": str(node.value)}}]}}
```

Comment:
Why `{"term": {"_collections": "HEP"}}`? This should work in general. Please, format these dictionaries in a way that they are easier to parse for humans :-)